table/tables/tables.go

```diff
@@ -1963,10 +1963,16 @@ type TemporaryTable struct {
 
 // TempTableFromMeta builds a TempTable from model.TableInfo.
 func TempTableFromMeta(tblInfo *model.TableInfo) tableutil.TempTable {
+	alloc := autoid.NewAllocatorFromTempTblInfo(tblInfo)
+	// Rebase the allocator if the base is specified.
+	if tblInfo.AutoIncID > 1 {
+		// We can omit the error because it won't return an error.
+		_ = alloc.Rebase(context.Background(), tblInfo.AutoIncID-1, false)
```

Comment:
Note: It's not easy to return an error here. It needs to update lots of interfaces because outer interfaces don't expect errors either.